packages/base/markdown.gts

```diff
@@ -14,13 +14,203 @@ class View extends Component<typeof MarkdownField> {
+      /* Subscript */
+      /** Must use <sub> **/
+
+      /* Superscript */
+      /** Must use <sup> **/
+
+      /* Blockquote */
+      .markdown-content :deep(blockquote) {
+        margin-top: var(--boxel-sp-lg);
+        margin-bottom: var(--boxel-sp-lg);
+        margin-right: auto;
+        margin-left: auto;
+        padding-top: var(--boxel-sp-4xs);
+        padding-bottom: var(--boxel-sp-4xs);
+        border-right: 1px solid black;
+        border-left: 1px solid black;
+      }
+      .markdown-content :deep(blockquote p) {
+        font-size: 1.5em;
+        font-style: italic;
+        margin-inline-start: var(--boxel-sp-xl);
+        margin-inline-end: var(--boxel-sp-xl);
+      }
+
+      /* Horizontal rule */
+      .markdown-content :deep(hr) {
+        border-bottom: none;
+        border-right: none;
+        border-left: none;
+        border-top: var(--boxel-border);
+      }
+
+      /* Code */
+      .markdown-content :deep(code) {
+        font-family: var(--markdown-code-font-family, monospace);
       }
+
+      /* Code Block */
       .markdown-content :deep(pre) {
         white-space: var(--boxel-markdown-field-pre-wrap, pre-wrap);
+        background-color: #efefef;
+        border-radius: var(--boxel-border-radius-xl);
+        padding: var(--boxel-sp-lg);
+      }
+
+      /* Link */
+      .markdown-content :deep(a),
+      .markdown-content :deep(a:hover) {
+        color: currentColor;
+        text-decoration: underline;
+      }
+
+      /* Image */
+      .markdown-content :deep(figure, img, svg) {
+        max-width: 100%;
+      }
+      .markdown-content :deep(figure) {
+        margin-top: var(--boxel-sp-lg);
+        margin-bottom: var(--boxel-sp-lg);
+        margin-right: auto;
+        margin-left: auto;
+      }
+      .markdown-content :deep(figcaption) {
+        font-size: 0.8125em;
+        font-style: italic;
+      }
+      .markdown-content :deep(img) {
+        border-radius: var(--boxel-border-radius-lg);
+        overflow: hidden;
+      }
+
+      /* Ordered & Unordered List */
+      .markdown-content :deep(ol),
+      .markdown-content :deep(ul) {
+        padding-left: 1.375em;
+        margin-top: var(--boxel-sp);
+        margin-bottom: var(--boxel-sp);
+        font-size: inherit;
+        font-weight: 400;
+        font-family: inherit;
+      }
+      /* Nested list */
+      .markdown-content :deep(ol ol),
+      .markdown-content :deep(ol ul),
+      .markdown-content :deep(ul ul),
+      .markdown-content :deep(ul ol) {
+        margin-top: var(--boxel-sp-xxxs);
+        margin-bottom: var(--boxel-sp-xxxs);
+      }
+
+      /* Task List */
+      .markdown-content :deep(ul:has(input[type='checkbox'])) {
+        list-style-type: none;
+        padding-left: 0;
+      }
+
+      /* Definition List */
+      /* Must use <dl> <dt> <dd> tags -- default browser styling */
+
+      /* Footnote */
+      /* Not available */
+
+      /* Emoji */
+      /* Must copy/paste emoji */
```

Comment:
What are these empty sections?

Reply:
Some markdown editors allow more syntax (https://www.markdownguide.org/cheat-sheet/) for stuff like definition lists, subscript/superscript, emojis, etc. Ours doesn't, but for most it's possible to use html inside the markdown field for it so I made notes.